src/ngraph/frontend/onnx_import/op/supported_ops.md

```diff
@@ -0,0 +1,137 @@
+## Supported Ops:
+
+Generally nGraph doesn't support tensors of types:
+
+* `string`,
+* `float16`,
+* `complex64`,
+* `complex128`.
+
+Value in `()` _brackets_ means that this op is available since specific opset version.
+Values seperated by `-` _dash_ represents opset versions introducing some changes. 
```

Comment:
Values separated by `-` _dash_ indicate that changes were made to that op, and that (it will |or| implementation will?) differ between versions.

Reply:
Updated comment - lets see if it's clearer now.